src/sentry/api/endpoints/project_performance_issue_settings_admin.py

```diff
@@ -0,0 +1,61 @@
+from rest_framework import serializers, status
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry import features, projectoptions
+from sentry.api.base import region_silo_endpoint
+from sentry.api.bases.project import ProjectEndpoint
+from sentry.api.endpoints.project_performance_issue_settings import SETTINGS_PROJECT_OPTION_KEY
+from sentry.api.permissions import SuperuserPermission
+
+SETTINGS_PROJECT_OPTION_ADMIN_KEY = "sentry:performance_issue_settings_admin"
+
+
+class ProjectPerformanceIssueAdminSettingsSerializer(serializers.Serializer):
+    uncompressed_assets_detection_enabled = serializers.BooleanField(required=False)
+    consecutive_http_spans_detection_enabled = serializers.BooleanField(required=False)
+    large_http_payload_detection_enabled = serializers.BooleanField(required=False)
+    n_plus_one_db_queries_detection_enabled = serializers.BooleanField(required=False)
+    n_plus_one_api_calls_detection_enabled = serializers.BooleanField(required=False)
+    db_on_main_thread_detection_enabled = serializers.BooleanField(required=False)
+    file_io_on_main_thread_detection_enabled = serializers.BooleanField(required=False)
+    consecutive_db_queries_detection_enabled = serializers.BooleanField(required=False)
+    large_render_blocking_asset_detection_enabled = serializers.BooleanField(required=False)
+    slow_db_queries_detection_enabled = serializers.BooleanField(required=False)
+
+
+@region_silo_endpoint
+class ProjectPerformanceIssueAdminSettingsEndpoint(ProjectEndpoint):
+    permission_classes = (SuperuserPermission,)  # type: ignore
```

Comment:
Can you please explain why the permission is limited to superuser? From the endpoint name sounds like something we want to open to org admins/owners?

Reply:
Adding onto this I see `admin` and `superuser` seem to be used interchangeably (eg: `ProjectPerformanceIssueAdminSettingsEndpoint` class name w/ superuser permission reqs). We are currently working in the process of setting API design standards, and some of those standards include:
- having the be URL representative of the resource hierarchy
- having an API do only one thing that can be guessed intuitively from url

If the endpoint is intended to be used for superusers only, would it be possible to replace references to admin with superuser in the file name, class names, and URL to reflect that?